oscar/Courses-AT05/src/main/java/courses/sampleapp/ui/RegisterPage.java

```diff
@@ -0,0 +1,105 @@
+package courses.sampleapp.ui;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by Administrator on 11/17/2017.
+ */
+public class RegisterPage extends BasePageObject {
+
+    @FindBy(id = "user_first_name")
+    WebElement firstName;
+
+    @FindBy(id = "user_last_name")
+    WebElement lastName;
+
+    @FindBy(id = "user_email")
+    WebElement email;
+
+    @FindBy(id = "user_password")
+    WebElement password;
+
+    @FindBy(id = "btn-signup")
+    WebElement btnSignUp;
+
+    /**
+     * Constructor
+     */
+    public RegisterPage() {
+      super();
+    }
+
+  /**
+   * This method set the values for register user.
+   *
+   * @param firstName as a String
+   * @param lastName as a String
+   * @param email as a string.
+   * @param password as a string.
+   * @return HomePage.
+   */
+  public HomePage registerUser(String firstName, String lastName, String email, String password) {
+    setFirstName(firstName);
+    setLastName(lastName);
+    setEmailUser(email);
+    setPasswordUser(password);
+    return clickSingUpButton();
+  }
```

Comment:
because pass several parameters if you already have the class User, sojo uses that class

Reply:
I will take into account